docs/Bigtable-batchsource.md

```diff
@@ -61,18 +61,18 @@ Data Type Mappings from BigQuery to CDAP
 The following table lists out different BigQuery data types, as well as the 
 corresponding CDAP data type for each BigQuery type.
 
-| BigQuery type | CDAP type   | Precision / Range           |
-|---------------|-------------|-----------------------------|
-| array         | unsupported |                             |
-| bool          | boolean     |                             |
-| bytes         | bytes       |                             |
-| date          | date        | YYYY-MM-DD                  |
-| datetime      | string      |                             |
-| float64       | double      | 8 bytes, double precision   |
-| geo           | unsupported |                             |
-| int64         | long        | 8 bytes                     |
-| numeric       | decimal     | 38 digits, 9 decimal places |
-| string        | string      |                             |
-| struct        | record      |                             |
-| time          | time        | microseconds                |
-| timestamp     | timestamp   | microseconds                |
\ No newline at end of file
+| BigQuery type | CDAP type                             |
+|---------------|---------------------------------------|
+| array         | unsupported                           |
```

Comment:
did you see my comment about this? It looks like the code is looking for repeat fields and treating them as arrays. Did you try it out and see a failure with arrays?